agent/xds/rbac.go

```diff
@@ -205,18 +216,47 @@ func removePermissionPrecedence(perms []*rbacPermission, intentionDefaultAction
 		}
 
 		perm.ComputedPermission = perm.Flatten()
+
+		//attach jwt permissions
+		perm.ComputedPermission = perm.GenerateJWTPermissions()
 		out = append(out, perm)
 	}
 
 	return out
 }
 
+func (p *rbacPermission) GenerateJWTPermissions() *envoy_rbac_v3.Permission {
+	if p.jwtInfos == nil {
```

Comment:
minor: The following suggestion should cover if the list is nil or empty

```suggestion
	if len(p.jwtInfos) == 0 {
```